deno/lib/types.ts

```diff
@@ -1032,11 +1084,11 @@ export class ZodString extends ZodType<string, ZodStringDef, string> {
           });
           status.dirty();
         }
-      } else if (check.kind === "jwt") {
-        if (!isValidJWT(input.data, check.alg)) {
+      } else if (check.kind === "ipRange") {
+        if (!isValidIPRange(input.data, check.cidr, check.version)) {
           ctx = this._getOrReturnCtx(input, ctx);
           addIssueToContext(ctx, {
-            validation: "jwt",
+            validation: "ipRange",
             code: ZodIssueCode.invalid_string,
             message: check.message,
           });
```

Comment:
[FYI] I just rearranged the order so that all ip-related code is grouped together for better readability. 
The internal code hasn't been changed 👍